examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/dem-packing-in-spouted-bed-generator.prm

```diff
@@ -0,0 +1,133 @@
+# Listing of Parameters
+#----------------------
+
+set dimension = 3
+
+#---------------------------------------------------
+# Simulation Control
+#---------------------------------------------------
+
+subsection simulation control
+  set time step        = 0.00001
+  set time end         = 1.00
+  set log frequency    = 1000
+  set output frequency = 10000
+  set output path      = ./output_dem/
+end
+
+#---------------------------------------------------
+# Timer
+#---------------------------------------------------
+
+subsection timer
+  set type = none
+end
+
+#---------------------------------------------------
+# Restart
+#---------------------------------------------------
+
+subsection restart
+  set checkpoint = true
+  set frequency  = 10000
+  set restart    = false
+  set filename   = dem
+end
+
+#---------------------------------------------------
+# Model parameters
+#---------------------------------------------------
+
+subsection model parameters
+  subsection contact detection
+    set contact detection method                = dynamic
+    set dynamic contact search size coefficient = 0.9
+    set neighborhood threshold                  = 1.3
+  end
+  subsection load balancing
+    set load balance method     = dynamic
+    set threshold               = 0.5
+    set dynamic check frequency = 10000
+  end
+  set particle particle contact force method = hertz_mindlin_limit_overlap
+  set particle wall contact force method     = nonlinear
+  set integration method                     = velocity_verlet
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+
+subsection lagrangian physical properties
+  set gx                       = 0
+  set gy                       = 0
+  set gz                       = -9.81
+  set number of particle types = 1
+  subsection particle type 0
+    set size distribution type            = uniform
+    set diameter                          = 0.005
+    set number of particles               = 100000
+    set density particles                 = 100 #2526 
+    set young modulus particles           = 1e6
+    set poisson ratio particles           = 0.25
+    set restitution coefficient particles = 0.97
+    set friction coefficient particles    = 0.4
+    set rolling friction particles        = 0.3
+  end
+  set young modulus wall           = 1e6
+  set poisson ratio wall           = 0.25
+  set restitution coefficient wall = 0.33
+  set friction coefficient wall    = 0.2
+  set rolling friction wall        = 0.3
+end
+
+#---------------------------------------------------
+# Insertion Info
+#---------------------------------------------------
+
+subsection insertion info
+  set insertion method                               = volume
+  set inserted number of particles at each time step = 100000
+  set insertion frequency                            = 2000
+  set insertion box minimum x                        = -0.075
```

Comment:
Same thing here: 

set insertion box coordinates.

Reply:
Were you using a recent version of Lethe? This parameter file should not run technically...